spec/requests/notifications_controller_spec.rb

```diff
@@ -228,6 +174,101 @@ def delete_notification(resp_code, matcher)
               read_high_priority.id
             ])
           end
+
+          it "should not bump last seen reviewable in readonly mode" do
+            SiteSetting.enable_experimental_sidebar_hamburger = true
+            user.update!(admin: true)
+            Fabricate(:reviewable)
```

Comment:
I think we can just `fab!` the `reviewable` once

Reply:
done!